extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -632,6 +637,19 @@ export class AmpStoryEmbeddedComponent {
         this.owners_.schedulePause(this.storyEl_, embedEl);
       }
     });
+
+    this.win_.addEventListener('keyup', event => {
+      if (
+        event.key === Keys.ESCAPE &&
+        this.state_ === EmbeddedComponentState.EXPANDED
+      ) {
+        event.preventDefault();
```

Comment:
Is this needed?

Reply:
I pictured it might be needed since we use it in the [bookend](https://github.com/ampproject/amphtml/blob/d1a7ee5a00f9e4cf08a06aeff321c97aa34c0517/extensions/amp-story/0.1/amp-story-bookend.js#L363-L364) and in the [share menu](https://github.com/ampproject/amphtml/blob/d1a7ee5a00f9e4cf08a06aeff321c97aa34c0517/extensions/amp-story/1.0/amp-story-share-menu.js#L196-L197).